api/controllers/game/scuttle.js

```diff
@@ -7,74 +7,68 @@ module.exports = function (req, res) {
-                const playerUpdates = {
-                  frozenId: null,
-                };
-                // Consolidate update promises into array
-                const updatePromises = [
-                  // Include game record so it can be retrieved downstream
-                  game,
-                  // Updates to game record e.g. turn
-                  Game.updateOne(game.id)
-                    .set(gameUpdates),
-                  // Updates to player record i.e. frozenId
-                  User.updateOne(player.id)
-                    .set(playerUpdates),
-                  // Clear target's attachments
-                  Card.replaceCollection(target.id, 'attachments')
-                    .members([]),
-                  // Remove card from player's hand
-                  User.removeFromCollection(player.id, 'hand')
-                    .members([card.id]),
-                  // Remove target from opponent's points
-                  User.removeFromCollection(opponent.id, 'points')
-                    .members([target.id]),
-                  // Scrap cards
-                  Game.addToCollection(game.id, 'scrap')
-                    .members([
-                      ...attachmentIds,
-                      card.id,
-                      target.id,
-                    ]),
-                ];
-
-                return Promise.all(updatePromises);
-              } else {
-                return Promise.reject({message: "That card is frozen! You must wait a turn to play it."});
-              }
-            } else {
-              return Promise.reject({message: "You can only scuttle an opponent's point card with a higher rank point card, or the same rank with a higher suit. Suit order (low to high) is: Clubs < Diamonds < Hearts < Spades"});
-            }
-          } else {
-            return Promise.reject({message: "You can only scuttle a card your opponent has played for points"});
-          }
-        } else {
-          return Promise.reject({message: "You can only play a card that is in your hand"});
-        }
-      } else {
+      if (game.turn % 2 !== player.pNum) {
         return Promise.reject({message: "It's not your turn."});
       }
+      if (card.hand !== player.id) {
+        return Promise.reject({message: "You can only play a card that is in your hand"});
+      }
+      if (target.points !== opponent.id) {
+        return Promise.reject({message: "You can only scuttle a card your opponent has played for points"});
+      }
+      if (card.rank < target.rank || (card.rank === target.rank && card.suit < target.suit)) {
+        return Promise.reject({message: "You can only scuttle an opponent's point card with a higher rank point card, or the same rank with a higher suit. Suit order (low to high) is: Clubs < Diamonds < Hearts < Spades"});
+      }
+      if (player.frozenId === card.id) {
+        return Promise.reject({message: "That card is frozen! You must wait a turn to play it."});
+      }
```

Comment:
I know I had asked you to try refactoring these to use one-line syntax, but actually looking at it again I think this is good as it is. It'll be hard to read super long lines that include long error messages and also the conditions. So this is good as it is